Source/WebInspectorUI/UserInterface/Views/MultiSidebar.js

```diff
@@ -322,7 +364,48 @@ WI.MultiSidebar = class MultiSidebar extends WI.Sidebar
     {
         this.dispatchEventToListeners(WI.Sidebar.Event.WidthDidChange, {sidebar: event.target, newWidth: event.data.newWidth});
     }
+    
+    _recalculateHeight(newHeight)
+    {
+        console.assert(newHeight);
+        let maximumHeight = this.element.parentElement.offsetHeight - WI.TabBrowser.MinimumHeight;
+        
+        // Need to add 1 because of the 1px border-top.
+        newHeight = Math.ceil(Number.constrain(newHeight, this.minimumHeight + 1, maximumHeight));
+        this.element.style.height = `${newHeight}px`;
 
+        if (this.collapsed)
+            return;
+        
+        for (let sidebar in this.sidebars) {
+            sidebar.selectedSidebarPanel?.needsLayout(WI.View.LayoutReason.Resize);
+        }
+
+        this.dispatchEventToListeners(WI.MultiSidebar.Event.HeightDidChange, {newHeight});
+    }
+    
+    _adjustPosition()
+    {
+        const isNarrow = WI.layoutMode === WI.LayoutMode.Narrow;
+        this.element.classList.toggle("on-bottom", isNarrow);
```

Comment:
NIT: I think just `"bottom"` would be fine.  you also could have `WI.updateLayoutMode` apply a `"narrow"` CSS class to the `<body>` instead and have this CSS be something like `body.narrow .multi-sidebar` (similar to how we have `body.docked.bottom`, `body.undocked`, `body[dir="rtl"]`, etc.)

Reply:
Did the latter.